client/rpcserver/types.go

```diff
@@ -189,10 +190,19 @@ func parseNewWalletArgs(params *RawParams) (*newWalletForm, error) {
 		appPass:    params.PWArgs[0],
 		walletPass: params.PWArgs[1],
 		assetID:    uint32(assetID),
-		account:    params.Args[1],
+	}
+	if len(params.Args) > 1 {
+		req.config, err = config.Parse([]byte(params.Args[1]))
+		if err != nil {
+			return nil, fmt.Errorf("config parse error: %v", err)
+		}
 	}
 	if len(params.Args) > 2 {
-		req.configText = params.Args[2]
+		cfg := make(map[string]string)
+		err := json.Unmarshal([]byte(params.Args[2]), &cfg)
```

Comment:
`cfg` map settings aren't making it into `req.config`.  Might also be good to log any overridden settings.  e.g.

```go
for key, val := range cfg {
	if fileVal, found := req.config[key]; found {
		log.Infof("Overriding wallet setting %s=%s with %s", key, fileVal, val)
	}
	req.config[key] = val
}
```

Reply:
Whoops. I could swear I tested this. Not well enough, obviously.